src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -83,6 +82,7 @@ import qualified Text.Parsec as Parsec
 import qualified Turtle hiding (encodeString)
 import qualified Turtle.Compat as Turtle (encodeString)
 import           Turtle                         (FilePath, (</>), (<.>))
+import NeatInterpolation qualified as Neat
```

Comment:
Why did you change just this one import to use `ImportQualifiedPost`, but not the others?